Sources/SwiftDocC/Infrastructure/DocumentationContext.swift

```diff
@@ -2221,8 +2227,8 @@ public class DocumentationContext: DocumentationContextDataProviderDelegate {
         for article in tutorialArticles {
             hierarchyBasedResolver.addTutorialArticle(article)
         }
-        for technology in technologies {
-            hierarchyBasedResolver.addTechnology(technology)
+        for tutorialTableOfContents in tutorialTableOfContents {
```

Comment:
This looks wrong, or at least very confusing.

Reply:
I could only find these two places (this and [line 664](https://github.com/swiftlang/swift-docc/pull/1039#discussion_r1796066921))  that do `for element in tutorialTableOfContents { ... }` and I find the minor awkwardness of the naming worth it overall for the big improvement in clarity of untying the code terminology and user-facing terminology.

Reply:
With the changes above, this would read:

```
for tutorialTableOfContents in tutorialTableOfContentsCollection {
```